docs/src/routes/docs/components/button.svx

```diff
@@ -0,0 +1,127 @@
+---
+shadow1: '0 2px 5px rgba(0, 0, 0, .14), 0 1px 5px rgba(0, 0, 0, .12);'
+shadowRaised: '0 2px 4px -1px rgba(0, 0, 0, .2), 0 3px 5px 0 rgba(0, 0, 0, .14), 0 1px 10px 0 rgba(0, 0, 0, .12);'
+---
+
+<script>
+  import { Button, Label } from 'attractions';
+  import { CheckIcon, XIcon } from 'svelte-feather-icons';
+  import H2 from 'src/mdsvex/h2.svelte';
```

Comment:
Why do you need to import it here? shouldn't mdsvex automatically replace any `h2` with this `H2`?

Reply:
It would replace `##` with a `<H2>`, but in this case we need to pass props, so I use the component explicitly.

Reply:
Another option would be to use `##` and auto-generate the id from the heading text itself, since I don't think there is a heading (at least h1 - h3) that doesn't deserve its fragment

Reply:
Yep, I wish that was possible, but Svelte currently doesn't allow accessing slots programmatically